specs/protocol/system-config.md

```diff
@@ -163,3 +163,9 @@ The contained log events are filtered and processed as follows:
 Note that individual derivation stages may be processing different L1 blocks,
 and should thus maintain individual system configuration copies,
 and apply the event-based changes as the stage traverses to the next L1 block.
+
+Logs for all transaction types are parsed for `SystemConfig` events, also from possibly unknown transactions types like
+those that are considered invalid batcher transaction types, see [L1 Retrieval][derivation.md#l1-retrieval]. However,
+they are parsed as if they were receipts for one of the known transactions types `0`, `1`, `2`, `3`, `4` and `0x7e`. If
+they fail this best-effort decoding process, they are dropped and considered invalid.
+The intention of this best-effort decoding is to future-proof the protocol for new L1 transaction types.
```

Comment:
I'm not sure if this "best-effort" specification is too vague and not precise enough. It seems like this could be too implementation-dependent...

Reply:
I've completely rewritten the spec and came up with a process that feels much more clearly defined.